### DISCUSSION TITLE
Unable to see my merged integration

### DISCUSSION THREAD

**Discussion by chakka-guna-sekhar-venkata-chennaiah** [fetchai/uAgents#500] [Integrations]
Hi Team, I'm unable to see my merged integration under the under the integration foler. I'm attaching the proof of image for reference 
![Screenshot_2024-08-11-16-04-59-693_com android chrome-edit](https://github.com/user-attachments/assets/13c40df9-d874-45a8-88ae-b3837bcedae0)

**Comment by abhifetch** [comment]
In line with our new standards and guidelines, we no longer push integrations that involve RapidAPI calls. We prioritize sources that are trustworthy and authentic. Additionally, for an integration to be classified as such under Fetch's terminology, it must consist of a single agent. The integration you've created is currently classified as a solution.

I suggest you include this in the following path: [Hackathons](https://github.com/fetchai/uAgents/tree/main/integrations/hackathons) .

If you have any further questions, feel free to reach out to us here.
